eap-xp2/image.yaml

```diff
@@ -0,0 +1,135 @@
+schema_version: 1
+
+name: "jboss-eap-7/eap-xp2-openjdk11-openshift-rhel8"
+description: "Red Hat JBoss Enterprise Application Platform XP 2.0 OpenShift container image."
+version: "2.0"
+from: "registry.redhat.io/ubi8/ubi:latest"
+labels:
+    - name: "com.redhat.component"
+      value: "jboss-eap-7-xp2-openjdk11-openshift-rhel8-container"
+    - name: "io.k8s.description"
+      value: "Platform for building and running JavaEE applications on JBoss EAP XP 2.0"
+    - name: "io.k8s.display-name"
+      value: "JBoss EAP XP"
+    - name: "io.openshift.expose-services"
+      value: "8080:http"
+    - name: "io.openshift.tags"
+      value: "builder,javaee,eap,eap7,eap-xp"
+    - name: "io.openshift.s2i.scripts-url"
+      value: "image:///usr/local/s2i"
+    - name: "maintainer"
+      value: "Red Hat"
+envs:
+    - name: HTTPS_ENABLE_HTTP2
+      value: "true"
+    - name: SCRIPT_DEBUG
+      description: If set to true, ensures that the bash scripts are executed with the -x option, printing the commands and their arguments as they are executed.
+      example: "true"
+ports:
+    - value: 8443
+modules:
+      repositories:
+          - name: cct_module
+            git:
+                  url: https://github.com/jboss-openshift/cct_module.git
+                  ref: 0.39.0
+          - name: jboss-eap-modules
+            git:
+                  url: https://github.com/jboss-container-images/jboss-eap-modules.git
+                  ref: EAP_XP_2_CR1
+          - name: jboss-eap-7-image
+            git:
+                  url: https://github.com/jboss-container-images/jboss-eap-7-image.git
+                  ref: EAP_XP_2_CR1_1
+          - name: wildfly-cekit-modules
+            git:
+                  url: https://github.com/wildfly/wildfly-cekit-modules.git
+                  ref: 0.18.8
+      install:
+          - name: jboss.container.openjdk.jdk
+            version: "11"
+          - name: jboss.container.maven.35.bash
+            version: "3.5"
+          - name: eap-xp-20-env
+            version: "1.0.0"
+          - name: jboss.container.eap.setup
+          - name: eap-install-cleanup
+          # This one indirectly installs common logging by creating a link to $BIN_HOME/bin/launch
+          # so must be after jboss.container.eap.cd
+          - name: jboss.container.maven.default.bash
+          - name: dynamic-resources
+          - name: jboss.container.eap.s2i.galleon
+          - name: jboss.container.eap.galleon
+          - name: jboss.container.eap.galleon.config.mp
+          - name: jboss.container.eap.galleon.eap73-compat
```

Comment:
This module is only required for 1.x and 7.3.x, it should be removed.

Reply:
fixed, thanks!